trinity/constants.py

```diff
@@ -11,18 +11,20 @@
 from p2p.kademlia import Address, Node
 
 
+# sync modes
 SYNC_FULL = 'full'
 SYNC_LIGHT = 'light'
 
+# lahja endpoint names
 MAIN_EVENTBUS_ENDPOINT = 'main'
 NETWORKING_EVENTBUS_ENDPOINT = 'networking'
 
-MAXIMUM_OBJECT_MEMORY_BYTES = 10000000
-
+# Network IDs: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
```

Comment:
This link is to the list of chain ids. (notably, Ethereum Classic is on network ID 1 and chain ID 61)

I don't know of a better list of network IDs than this: https://ethereum.stackexchange.com/a/17101/1461